scalableminds/webknossos
# Try importing every file as nml and protobuf before failing

This PR solves multiple small issues. 
#4372 improves the import to try both, nml and protobuf, import in tracing views.
#4379 fixes opening view only dataset links in arbitrary mode.
#4327 Adds a loading indication to the reload button in the dataset actions of a dataset in the dashboard

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
For #4372:
- Open a skeleton tracing.
- Try dropping a valid NML file with the extensions `.nml`, `.xml` and `.some_other_format`. Every single import should work.
- Do the same with a valid protobuf file.
- Try importing an invalid document like a `pdf` file. An error toast should pop up.

For #4379:
- Open a dataset in only view mode, switch to any arbitrary mode.
- Then take the link from the current tab and open it in another tab. The initial view mode should be the arbitrary view mode you chose before.

For #4327:
> This disables the edit and reload link/button in the dataset actions in the dashboard while one reload is active.

- please also check whether this is working.

### Issues:
- fixes #4372, #4379, #4327

------
